packages/block-editor/src/store/selectors.js

```diff
@@ -2972,3 +2973,33 @@ export const isGroupable = createRegistrySelector(
 			);
 		}
 );
+
+export const getSectionsContainerClientId = createRegistrySelector(
+	( select ) =>
+		( state, postType = 'page' ) => {
+			const CONTENT_TYPES = [ 'post', 'page' ];
+
+			if ( CONTENT_TYPES.includes( postType ) ) {
+				const postContentBlocks = getBlocksByName(
+					state,
+					'core/post-content'
+				);
+
+				if ( postContentBlocks?.length > 0 ) {
+					return postContentBlocks[ 0 ];
+				}
+			}
```

Comment:
I discussed this with @richtabor...

If you are assembling a page then the `core/post-content` block represents the "section container" of the Page.

That is very different to a Template where it's likely the Group with the `<main>` tag is the the section container.

Reply:
What does this mean?

Reply:
A template may not have a `core/post-content` block, but rather a `<main>` element `core/group` block. 

When invoking zoom out from a page/post (not the template editor), there will be `core/post-content` block.